refresher/pygmt_tutorial.ipynb

```diff
@@ -0,0 +1,646 @@
+    "        # Plot the original data\n",
+    "        fig.plot(data=data, region=region, projection=\"M?\", style=\"c1p\", cmap=True)\n",
+    "    # Use the fig.set_panel method to select the second panel\n",
+    "    with fig.set_panel(panel=1):\n",
+    "        # Plot the data returned from pygmt.blockmedian\n",
+    "        fig.plot(data=data_median, region=region, projection=\"M?\", style=\"c3p\", cmap=True)\n",
+    "    # Use the fig.set_panel method to select the third panel\n",
+    "    with fig.set_panel(panel=2):\n",
+    "        # Plot the data returned from pygmt.surface\n",
+    "        fig.grdimage(grid=grid, region=region, projection=\"M?\")\n",
+    "    # Use the fig.set_panel method to select the fourth panel\n",
+    "    with fig.set_panel(panel=3):\n",
+    "        # Plot the earth relief data provided by PyGMT\n",
+    "        fig.grdimage(grid=relief, region=region, projection=\"M?\")\n",
+    "# Add a colorbar to the figure with the label \"Elevation (m)\"\n",
+    "# Position the colorbar such that the center, middle (`+jCM`) of the colorbar\n",
+    "# is 7.5 cm in the x-direction and -1 cm in the y=direction from the\n",
+    "# bottom-left corner of the plot (`+x7.5c/-1c`\n",
+    "# Make the colorbar horizontal (`+h`) and 6 cm wide (`+w6c`)\n",
+    "fig.colorbar(frame='x+l\"Elevation (m)\"', position=\"x7.5c/-1c+jCM+w6c+h\")\n",
+    "fig.show()\n",
+    "\n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9c1ce253-1b4f-4d96-97e9-7b6b43b07d09",
+   "metadata": {},
+   "source": [
+    "## 5.1 Configure PyGMT settings\n",
+    "\n",
+    "PyGMT settings can be customized using [pygmt.config](https://www.pygmt.org/dev/api/generated/pygmt.config.html#pygmt.config).\n",
+    "More instruction is offered in the [configuring PyGMT defaults tutorial](https://www.pygmt.org/dev/tutorials/configuration.html#sphx-glr-tutorials-configuration-py).\n",
+    "The full list of configurable settings can be found in the [GMT configuration documentation](https://docs.generic-mapping-tools.org/latest/gmt.conf.html)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "58ba6ffd-d6f0-490d-be49-4f3fd054d622",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create two coastline plots\n",
+    "fig = pygmt.Figure()\n",
+    "fig.coast(frame=True, region=region, projection=\"M7c\", land=\"darkgray\", water=\"skyblue\")\n",
+    "fig.shift_origin(xshift=\"10c\")\n",
+    "fig.coast(frame=True, region=region, projection=\"M7c\", land=\"darkgray\", water=\"skyblue\")\n",
+    "fig.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "64f65a85-5ab4-4c40-a93a-6cb2a7be146a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Change the font size for annotations to 20 points\n",
+    "# Change applies for remaining code\n",
+    "fig = pygmt.Figure()\n",
+    "pygmt.config(FONT_ANNOT_PRIMARY=\"20p\")\n",
```

Comment:
Need to be careful here, as the next cell will also have `FONT_ANNOT_PRIMARY="20p"` applied if you don't reset it. Maybe swap the order of this cell with the next?

Reply:
I'll double check in the morning, but for me this worked since there is a new figure created in the next cell. I can switch either way.

Reply:
> as the next cell will also have `FONT_ANNOT_PRIMARY="20p"` applied if you don't reset it.

It won't, because `pygmt.config` is called after the figure is created, so it only changes the figure-level settings. Any setting changes will only affect the current figure.

Reply:
Ah right, forgot about the nesting behaviour. All good to keep this as is then.